packages/client/bin/cli.ts

```diff
@@ -59,6 +59,10 @@ import type { CustomCrypto } from '@ethereumjs/common'
 import type { GenesisState } from '@ethereumjs/util'
 import type { AbstractLevel } from 'abstract-level'
 
+const promClient = require('prom-client')
```

Comment:
I know this is a little nitpicky but is there a specific motiviation for using `require` syntax here?  This just always ends up in painful places down the line since we will hopefully one day be ESM only and no longer ever need this.  You should be able to use `import * as promClient from 'prom-client'` here.  We do so in Ultralight and should hopefully work the same here.

Reply:
Good point. Fixed.